lib/data/multipleBackendGateway.js

```diff
@@ -215,9 +215,9 @@ const multipleBackendGateway = {
     },
 
     objectTagging: (method, key, bucket, objectMD, log, cb) => {
-        const location = objectMD.dataStoreName;
+        const location = objectMD.dataStoreName || undefined;
```

Comment:
I was thinking that if `objectMd` does not have the field `dataStoreName`, it will be `undefined` anyway.